app/src/main/java/com/example/fitjournal/home/presentation/components/card/CardioCard.kt

```diff
@@ -42,14 +39,23 @@ fun CardioCard(
                     )
             )
             Spacer(modifier = Modifier.height(Spacing.spacing8))
-            MostRecentWorkoutSession()
+            if (
+                cardioUi.distance != null &&
+                cardioUi.time != null
+            ) {
```

Comment:
I left a comment about moving null checks to an extension function inside data class, we can do that here